modules/CmpdReg/src/client/src/SearchResults.js

```diff
@@ -74,7 +74,7 @@ $(function() {
 
         newLot: function() {
 	        if ( !window.configuration.metaLot.saltBeforeLot && this.model.get('lotIDs').length==1) {
-		        this.trigger('newLot', this.model.get('lotIDs')[0].corpName);
+		        this.trigger('newLot', this.model.get('corpName'));
```

Comment:
Is doing this fine? I'm not sure if there would be repercussions elsewhere.

Reply:
As long as you've tested the New Lot button from within Search Results locally, and tested it with ACASLabelSequence and "standard" modes, I think you're all set to make this change.

Reply:
I tested with the search results locally with both ACASLabelSequence and "standard" modes so sounds good